CSSS/Kernel.cs

```diff
@@ -166,6 +171,32 @@ public bool PerformTasks()
                 PerformIssueCheckTasks();
             }
 
+            // If we should shutdown
+            if (shouldShutdown)
+            {
+                // Run the shutdown command
+                var process = new System.Diagnostics.Process();
+                process.StartInfo.FileName = "shutdown";
+                switch (config.operatingSystemType)
+                {
+                    case Config.OperatingSystemType.WinNT:
+                        // On Windows, we can specify the seconds
+                        // to delay the shutdown :D
+                        // So we shutdown in 15 seconds (should be suffficient time)
+                        process.StartInfo.Arguments = "-s -t 15";
+                        break;
+                    case Config.OperatingSystemType.Linux:
+                        // Linux doesn't let you specify seconds... >:(
+                        process.StartInfo.Arguments = "-h +1";

```

Comment:
Could the `+1` not be replaced by `now` for an immediate shutdown?

Reply:
Ideally a short period of time would be useful to ensure that the program isn't force closed before necessary. Perhaps further discussion/testing is needed regarding any potential "grace periods".